internal/test/e2e/kubectl.go

```diff
@@ -20,7 +21,27 @@ import (
 	"k8s.io/apimachinery/pkg/util/sets"
 )
 
-const testNameLabelKey = "e2e.etcd.improbable.io/test-name"
+const (
+	testNameLabelKey = "e2e.etcd.improbable.io/test-name"
+	// The etcd image containing etcdctl
+	etcdctlImage = "quay.io/coreos/etcd:v3.3.18"
+	// charSet defines the alphanumeric set for random string generation
+	charSet = "0123456789abcdefghijklmnopqrstuvwxyz"
+)
+
+var (
+	rnd = rand.New(rand.NewSource(time.Now().UnixNano()))
+)
+
+// randomString returns a random alphanumeric string.
+// Copied from https://github.com/kubernetes-sigs/cluster-api/blob/v0.2.9/util/util.go#L68
+func randomString(n int) string {
+	result := make([]byte, n)
```

Comment:
This only *happens* to work because all the characters in `charSet` can be encoded to a single byte in UTF-8. Should this be `[]rune`?

Reply:
I added a comment to the charSet const, as discussed in person.